sup3r/models/surface.py

```diff
@@ -0,0 +1,526 @@
+# -*- coding: utf-8 -*-
+"""
+Special models for surface meteorological data.
+"""
+import logging
+from fnmatch import fnmatch
+import numpy as np
+from PIL import Image
+from warnings import warn
+from sup3r.models.abstract import AbstractSup3rGan
+
+logger = logging.getLogger(__name__)
+
+
+class SurfaceSpatialMetModel(AbstractSup3rGan):
```

Comment:
Do you think it's worth including a `train` method here? Would it be useful merely for reproducing these coefficients?

Reply:
definitely, took some time but worth doing for sure.